packages/utils/src/misc.ts

```diff
@@ -510,3 +375,12 @@ export function addContextToFrame(lines: string[], frame: StackFrame, linesOfCon
     .slice(Math.min(sourceLine + 1, maxLines), sourceLine + 1 + linesOfContext)
     .map((line: string) => snipLine(line, 0));
 }
+
+/**
+ * Determines if tracing is currently enabled.
+ *
+ * Tracing is enabled when at least one of `tracesSampleRate` and `tracesSampler` is defined in the SDK config.
+ */
+export function hasTracingEnabled(options: Options): boolean {
+  return !!options.tracesSampleRate || !!options.tracesSampler;
```

Comment:
```suggestion
  return 'tracesSampleRate' in options || 'tracesSampler' in options;
```

It's more descriptive imo :)

Reply:
You're right. :-)

Fixed.